packages/docs/src/routes/docs/(qwik)/components/state/index.mdx

```diff
@@ -114,8 +115,7 @@ export default component$(() => {
       </button>
       <br />
       <button onClick$={() => {
-        // Because store is deep watched, this will trigger a re-render
-        store.list.push(`Item ${store.list.length}`);
+        store.list = [...store.list, `Item ${store.list.length}`]
```

Comment:
the previous example should work, withoyt copying the array, it's using a store.